coderd/database/migrations/000234_fix_notifications_user_created.down.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE notification_templates
```

Comment:
I don't really see value in restoring the typo in the down migration.

Reply:
just consistency?